### PR TITLE
Upgrade Charts to be compatible with Xcode 14

### PR DESCRIPTION
**Pull request by ljungqvist**

### CODE REVIEW

**Comment by JohnBerd** [comment]
We need this to be merged ASAP

File: react-native-charts-wrapper.podspec [hunk 1]
```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.homepage     = package["homepage"]
 
   s.license      = package["license"]
-  s.platform     = :ios, "9.0"
+  s.platform     = :ios, "13.0"
```
Comment (vismutls) [inline react-native-charts-wrapper.podspec:14]: Why do you need version 13? Charts requires at least version 12

**Comment by smadan** [comment]
@wuxudong please merge this as soon as possible.

**Comment by Ti-tanium** [comment]
@wuxudong Please merge this ASAP.

**Comment by jhonatancunha** [comment]
@wuxudong  Please merge this

**Comment by phamhieu1412** [comment]
@wuxudong merge and build package soon as pleass :((((

**Comment by Ti-tanium** [comment]
@wuxudong merge and build package soon as pleass :((((

**Comment by sanjeev-karat** [comment]
Can this be build as a package soon, please?

**Comment by wuxudong** [comment]
check v0.5.10

**Pull request merged**